tutorials/in-app/healthBar.json

```diff
@@ -0,0 +1,485 @@
+{
+  "id": "healthBar",
+  "editorSwitches": {
+    "Start": {
+      "editor": "Scene",
+      "scene": "level"
+    },
+    "SwitchToEvents1": {
+      "editor": "EventsSheet",
+      "scene": "level"
+    }
+  },
+  "availableLocales": [
+    "en",
+    "fr"
```

Comment:
I can add es & pt afterwards if you want

Reply:
It would be great!